README.md

```diff
@@ -67,6 +67,8 @@ Running the unittests can be done with the command:
 
 To run integration tests You will need to download and install chromedriver http://chromedriver.chromium.org/downloads
 
+You will need Gradle installed to run IRIDA
```

Comment:
You don't actually need gradle installed as the `./gradlew` will download it and then run it.

Reply:
Fixed.